### DISCUSSION TITLE
8 digit authentication code versus 6 digit

### DISCUSSION THREAD

**Discussion by Papa-26** [community/community#80824] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am trying to activate automatic authentication via CLI. after entering "gh auth login", and going through the steps for web form, I am asked for an 8 digit authentication code, but my Google authenticator provides only 6 digits from Github?

**Comment by Papa-26** [comment, marked as answer]
Issue resolved. Authentication code not via Google Authentication app.

**Reply by emmacabletech** [reply]
I have the same issue, what did you do to resolve it? where did you get the authentication code from?

**Reply by dwainm** [reply]
The Auth code will be in the CLI.

**Reply by kevinhellos** [reply]
Thank you for the reply. It helps me.

**Reply by ahmadSum1** [reply]
> The Auth code will be in the CLI.

This could be displayed on the page

**Comment by Nyololo** [comment]
I have the same issue

**Reply by EthanThePhoenix38** [reply]
I have it too and GH didn't provide any solution... 
where is that code!?!?

**Comment by subupatayach** [comment]
gmail, and all mail concert my mail id account acctiovets online support
and all world all contrary all bank accounts actiovtes on online 0balance
acctiovets and online international transaction money created my mail 🆔
accounts actiovtes and segunail tower setelight everything online include
support my mail 🆔 accounts on online.

**Comment by rcirjak** [comment]
please just copy it to clipboard on gh auth login run

**Reply by MirkoCizmovic** [reply]
its in CMD, copy your One-time code before you open link in browser

**Comment by cjpanilag** [comment]
BOGO MO'NG TANAN! 8 digit code. IDK how to locate this shits!

**Comment by iheb-draouil** [comment]
If anyone's trying to get Cursor to work with GithHub via the UI, the code is not from your Authenticator app, it's from the Cursor editor itself (bottom left).

**Reply by DLTKRSuresh** [reply]
it is displayed on the windows command prompt  How would you like to authenticate GitHub CLI? Login with a web browser

! First copy your one-time code: XXXX-XXXX
